### PR TITLE
Feature/ MUI theme config

### PR DESCRIPTION
**Pull request by zimashima**

### CODE REVIEW

**Review by squashgray — APPROVED** [review]
nice colors!

**Review by danimal-johnson — APPROVED** [review]
Looks good!

File: src/theme/typography.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const typography = {
+  fontFamily: [
+    '-apple-system',
+    'BlinkMacSystemFont',
+    '"Segoe UI"',
```
Comment (allie-rae) [inline src/theme/typography.ts:5]: Some of these have two sets of quotes

File: src/theme/palette.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { sniffrPrimaryColor, sniffrSecondaryColor } from './custom';
+
+const palette = {
+  primary: {
+    main: sniffrPrimaryColor,
```
Comment (allie-rae) [inline src/theme/palette.ts:5]: Do these colors need to be defined & imported? You can access them through the theme later, so I feel like putting them in the theme is defining them well enough. 🤔

**Review by allie-rae — APPROVED** [review]
Wooo! 🥰

File: src/theme/palette.ts (hunk 2 above)
Comment (allie-rae) [reply src/theme/palette.ts:5]: Also: this is a nitpick, feel free to ignore. :3

**Pull request merged**